ingestion_tools/scripts/importers/db/dataset.py

```diff
@@ -42,7 +42,7 @@ def get_direct_mapped_fields(cls) -> dict[str, Any]:
             "last_modified_date": ["dates", "last_modified_date"],
             "related_database_entries": ["cross_references", "related_database_entries"],
             "related_database_links": ["cross_references", "related_database_links"],
-            "dataset_publications": ["cross_references", "dataset_publications"],
```

Comment:
Just as a safety measure, we should add a fallback here to check the old path if this is none.